### ISSUE TITLE
mistake with loss

### ISSUE THREAD

**Issue by Guocode** [shufanwu/colorNet-pytorch#3]
cross_entropy_loss = 1/300 * F.cross_entropy(class_output, classes)
torch.nn.functional has no implement of backward, so cross_entropy_loss will not drop in training, use nn.CrossEntropyLoss() instead.

**Comment by shufanwu** [comment]
Thanks for your reminding,  but actually, there have little difference between the two implementations of cross_entropy loss. you can see details in https://discuss.pytorch.org/t/f-cross-entropy-vs-torch-nn-cross-entropy-loss/25505
